.eslintrc.json

```diff
@@ -1,3 +1,19 @@
 {
-  "extends": "brightspace/lit-config"
+  "extends": "brightspace/lit-config",
+  "overrides": [{
+    "files": [
+      "**/demo/**/*.html"
+    ],
+    "rules": {
+      "no-console": 0
```

Comment:
Can we move this override into an `.eslintrc.json` inside the `demo` directory?

Reply:
Sure thing, was following pattern I saw in core library for rule exceptions - I have no issue moving spot though. Suggest updating pattern in core as well if this is the preference.

Reply:
Thanks. Yeah I wish core was set up the way this repo is -- with `demo/` and `test/` at the root instead of nested inside each component directory. This setup allows the `eslint` configs for demo/test/src to exist in just 1 place instead of copied into each sub-directory.